KUBEVIRT_LOCAL_TESTING.md

```diff
@@ -1,34 +1,53 @@
 # [kubevirtci](README.md): Testing kubevirt with locally provisioned cluster
 
-With the changes in place you also can directly execute [`make functest`](https://github.com/kubevirt/kubevirt/blob/master/docs/getting-started.md#testing) against a cluster locally with kubevirt that was provisioned using `kubevirtci`.
+After making changes to a kubevirtci provider, you should test it locally before publishing it.
+
+With the changes in place you can also directly execute [`make functest`](https://github.com/kubevirt/kubevirt/blob/master/docs/getting-started.md#testing) against a cluster locally with kubevirt that was provisioned using `kubevirtci`.
+
+Lets go on the required steps, starting by provisioning, all the way to `make functest`.
+Note: the `...` in the beginning of `.../directory_name` means the full path of `directory_name`.
 
 Steps:
 
 ### kubevirtci: provision cluster locally
 
 ```bash
 cd .../kubevirtci
-cd cluster-provision/k8s/1.17.0
+cd cluster-provision/k8s/1.19
 ../provision.sh      # this also calls cluster-up to check whether the cluster will really start and have the pods ready
```

Comment:
I don't follow. If this calls `cluster-up`, why do set the env variables only after?

Reply:
provision calls cluster-up just for sanity check the cluster it created, and then it cluster-down it.
The next paragraph is to spin your own cluster-up with the changes, test it deeper, 
or even test it with kubevirt (until this point we were for sure on kubevirtci, from this point we can move to kubevirt)

Reply:
Added `KUBEVIRTCI_LOCAL_TESTING.md` which is kubevirtci dedicated.